src/dataValidators/articlesValidator.js

```diff
@@ -1,25 +1,28 @@
 'use strict';
 
 const {HTTP_CODES} = require(`../service/constants`);
-const articleKeys = [`title`, `announce`, `fullText`, `category`, `comments`];
+const articleKeys = [`title`, `announce`, `fullText`, `categories`, `comments`];
 
 const validateArticleAttr = (req, res, next) => {
   const isExistArticleKeys = req.body && Object.keys(req.body);
   const isExistNewArticles = isExistArticleKeys.some((key) => articleKeys.includes(key));
 
   if (!isExistArticleKeys || !isExistNewArticles) {
     res.status(HTTP_CODES.BAD_REQUEST).send(`Bad request`);
+    return;
   }
```

Comment:
не люблю я пустые ретурны
```suggestion
  if (!isExistArticleKeys || !isExistNewArticles) {
    return res.status(HTTP_CODES.BAD_REQUEST).send(`Bad request`);
  }
```

Reply:
тут и далее аналогично
приходи на консультацию - обсудим